docs/src/refman/monitor.txt

```diff
@@ -23,6 +24,7 @@ typedef struct ALLEGRO_MONITOR_INFO
    int y1;
    int x2;
    int y2;
+   int dpi;
```

Comment:
We can't actually add stuff to this struct, for backwards compatibility reasons (e.g. imagine a game compiled for 5.2.4 then running on 5.2.5 with this argument added; the library will write to this byte offset while the binary only allocated enough for the 4 fields).

I think the best option here is to just make a new function.